bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/package.scala

```diff
@@ -38,6 +38,7 @@ package object immutable {
   type CloudCover = M.CloudCover
   object CloudCover extends EnumObject[M.CloudCover] {
     val BEST = M.CloudCover.cc50
+    val CC70 = M.CloudCover.cc70
```

Comment:
Are these selectable on the instrument selection UI?

Reply:
They are selectable in the conditions UI; we just didn't have them specifically linked to names because we weren't using them for anything previously.